lightwood/mixers/sklearn_mixer.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+
+from sklearn.svm import SVC
+from sklearn.linear_model import SGDClassifier, SGDRegressor, LinearRegression
+from sklearn.naive_bayes import GaussianNB
+from sklearn.metrics import r2_score, balanced_accuracy_score
+
+from lightwood.logger import log
+from lightwood.constants.lightwood import COLUMN_DATA_TYPES
+from lightwood.mixers import BaseMixer
+
+CLASSIFICATION_MODELS = [
+    (SGDClassifier, {}),
+    (SVC, {}),
+    (GaussianNB, {}),
+]
+
+REGRESSION_MODELS = [
+    (LinearRegression, {}),
+    (SGDRegressor, {}),
+]
```

Comment:
Any particular reason for the choice of models ? 

If so, please make some comments as to why.

It will fall to the "research" side of things to optimize this mixer anyway, so don't worry too much about picking "the best" models, but it's worth noting down the intutions/experiments that made you pick these one since it will ease @paxcema 's job.

Reply:
my focus was just on making this work, so I quickly chose a few popular regression and classification models